src/injector.js

```diff
@@ -313,5 +314,12 @@ window.FAI.bundleHost = window.FAI.bundleHost || '/';
     };
 
     NS.INJECTOR.readyState = 'ready';
-    document.dispatchEvent(new Event('injectorReady'));
+    /* Not supposed to need to do this for IE11, but apparently we do... */
+    if (document.createEvent) {
+        event = document.createEvent('Event');
+        event.initEvent('injectorReady', false, true);
+    } else {
```

Comment:
It is *currently* well-supported, but it is also deprecated and will not always be supported.